frontend/src/pages/enabledApplications/EnabledApplications.tsx

```diff
@@ -20,6 +21,16 @@ type EnabledApplicationsInnerProps = {
 const EnabledApplicationsInner: React.FC<EnabledApplicationsInnerProps> = React.memo(
   ({ loaded, loadError, components }) => {
     const isEmpty = !components || components.length === 0;
+
+    // fire an individual segment.io tracking event for every enabled application
+    if (!isEmpty) {
+      components.forEach((c) => {
+        fireTrackingEvent('Application Enabled', {
+          name: c.metadata.name,
+        });
+      });
+    }
+
```

Comment:
How often do we want this event? I would think only once so I would put this in a `React.useEffect` and only update  on `isEmpty` change.

Reply:
@jeff-phillips-18 What if there is already an enabled application and the user enables another one? Should we use `components` as the dependency?

Reply:
Correct, but not on every render. We could do it instead in the resource watcher 🤷‍♂️